src/worker.c

```diff
@@ -187,7 +187,7 @@ worker_main(Datum main_arg)
 		/* Wait 10 seconds */
 		WaitLatch(&MyProc->procLatch,
 					WL_LATCH_SET | WL_TIMEOUT | WL_EXIT_ON_PM_DEATH,
-					10000L,
+					200L,
```

Comment:
@steve-chavez this is the only non-test related change

Reply:
Do you think the worker should be called this often? Perhaps a 1 second interval would be better.
```suggestion
					1000L,
```

Reply:
Otherwise it will execute many `SELECT .. LEFT JOIN` queries too often.